src/lib/components/labeledInput/labeledInput.module.scss

```diff
@@ -79,6 +79,13 @@
     }
   }
 
+  input[readonly], input[readonly]:focus {
+    background: $grey-30;
```

Comment:
Same here, I prefer themefied values instead of the variable directly, because on different websites we have different background colors